Cargo.toml

```diff
@@ -25,14 +25,15 @@ hex-literal = "0.3"
 iso7816 = "0.1"
 serde = { version = "1", default-features = false, features = ["derive"] }
 trussed = { version = "0.1", features = ["serde-extensions"] }
-trussed-auth = { version = "0.2" }
+trussed-auth = { version = "0.3" }
 untrusted = "0.9"
 vpicc = { version = "0.1.0", optional = true }
 log = "0.4"
 heapless-bytes = "0.3.0"
 subtle = { version = "2", default-features = false }
 trussed-rsa-alloc = { version = "0.1.0", features = ["raw"], optional = true }
-trussed-staging = { version = "0.1.0", features = ["chunked", "encrypted-chunked"]}
+trussed-chunked = "0.1.0"
+trussed-staging = { version = "0.3.0", features = ["chunked"], default-features = false }
```

Comment:
`trussed-staging` should only be pulled in for the `virt` feature.  With `trussed-derive`, I don’t think we really need the `virt` module at all, but we can discuss that separately.

Reply:
Right, I made trussed-staging an optional dependency.

Using the derive would not allow removing altogether from the dependencies so it will be done in another PR.